docs/Addons.md

```diff
@@ -30,16 +28,15 @@ env:
 
 ### **Getting an OCM refresh token for your tests**
 
-You will need to request an OCM refresh token in order to run your tests. The easiest way to do this is to visit [https://cloud.redhat.com/openshift/token](https://cloud.redhat.com/openshift/token) and copy the OFFLINE_REFRESH_TOKEN. 
+You will need to request an OCM refresh token in order to run your tests. The easiest way to do this is to visit [https://cloud.redhat.com/openshift/token] and copy the OFFLINE_REFRESH_TOKEN. If you do not have an account or quota, please see [Managing Organization Quota]
 
 Your account will need the following permissions:
 
 *   Credentials API access
-*   ...
 
 ### **Configuring your job to use your OCM refresh token**
 
-In order to run addon tests in osde2e, you will need to create a secret in Origin CI with your OCM refresh token. Please follow [these instructions](https://github.com/openshift/release/blob/e877df16a32be22b60a62b6313ef3e0fe2e9256b/core-services/secret-mirroring/README.md) to both create a secret and a secret mapping into the ci namespace.
+In order to run addon tests in osde2e, you will need to create a secret in Origin CI with your OCM refresh token. Please follow [these instructions] to both create a secret and a secret mapping into the ci namespace.
```

Comment:
Let's take this opportunity to mention that the OCM refresh token needs to be named `ocm-refresh-token` as well.